test/inject.test.js

```diff
@@ -1,10 +1,10 @@
 import React, { Component } from "react"
 import * as PropTypes from "prop-types"
-import ReactDOM from "react-dom"
 import * as mobx from "mobx"
 import { action, observable } from "mobx"
 import { observer, inject, Provider } from "../src"
-import { createTestRoot, cleanupTestRoot, sleepHelper, withConsole } from "./index"
+import { sleepHelper, withConsole } from "./index"
```

Comment:
Can we perhaps use `jest-mock-console` instead?

Reply:
It has nothing to do with `react-testing-library`. I can investigate it in the separate PR.